core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor.html

```diff
@@ -25,6 +25,9 @@
     <div ng-if="areFallbacksEnabled()">
       {% include 'pages/exploration_editor/editor_tab/state_editor_fallbacks.html' %}
     </div>
+    <div>
+      {% include 'pages/exploration_editor/editor_tab/state_editor_hints.html' %}
+    </div>
```

Comment:
So I was thinking about this, maybe leaving it on the Editor page by default will make it more noticeable to the creator and that they'd tend to use it more? I think the fallbacks feature wasn't used as much because it wasn't on the editor page by default. Not sure though.